File: src/Client/inventory-track/src/app/models/dto/report/report-response-dto.ts
// src/app/models/dto/report/report-response-dto.ts
import {ReportType} from "./enums/report-type";
import {DateSelect} from "./enums/data-select";

export interface ReportResponseDto {
  id: string;
  name: string;
  reportType: ReportType;
  dateSelect: DateSelect;
  createdAt: string;
}

File: src/Services/ReportService/ReportService.Application/DTOs/Response/Report/ReportResponseDto.cs
using ReportService.Domain.Enums;

namespace ReportService.Application.DTOs.Response.Report;

public class ReportResponseDto
{
    public Guid Id { get; set; }
    public string Name { get; set; }
    public ReportType ReportType { get; set; }
    public DateSelect DateSelect { get; set; }
    public DateTime CreatedAt { get; set; }
    public Guid CompanyId { get; set; }
}